### DISCUSSION TITLE
Detecting the voices with low energy

### DISCUSSION THREAD

**Discussion by zyb8543d** [snakers4/silero-vad#373] [Q&A]
For relatively small sounds, it cannot be detected. Can these small sounds be detected by adjusting the parameters? For these small sounds, I can do it via webrtc vad mode=2.

**Comment by snakers4** [comment]
If small means quiet, then you may plot the probability plot (`visualize_probs=True`), and tweak the most obvious params here:

https://github.com/snakers4/silero-vad/blob/563106ef8cfac329c8be5f9c5051cd365195aff9/utils_vad.py#L164-L175

**Reply by zyb8543d** [reply]
the small not means quiet and the voices are small but sounds clear. it doesn't seem to work for weaking the most obvious params.
